enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -0,0 +1,339 @@
+---
+title: windows-node-cluster-wide-egress-proxy
+authors:
+  - "@saifshaikh48"
+reviewers:
+  - "@openshift/openshift-team-windows-containers"
+  - "@openshift/openshift-team-network-edge, for general approach to proxy config consumption, motivation, risks, and testing"
+approvers:
+  - "@aravindhp"
+api-approvers:
+  - None
+creation-date: 2023-02-16
+last-updated: 2023-03-06
```

Comment:
You don't need to update for each review request. I think this should match for the first version. `last-updated` will change on future updates.